clarify experimental wording (#22425)
mdn/content: files/en-us/mdn/writing_guidelines/experimental_deprecated_obsolete/index.md

- It is implemented and enabled by default in _less than two_ modern major browser rendering engines.
- Its defining specification is likely to change significantly in backwards-incompatible ways (i.e., it may break existing code that relies on the feature).

If one or both of these conditions is true, then you should think carefully before you start using that technology in any kind of production project (i.e., a project that is not just a demo or experiment). You are however encouraged to try it out and provide feedback to browser vendors and standards authors. <!--taken from https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#status-information -->

Conversely, a technology is no longer considered **experimental** if one or more of the following conditions is true:

- It is implemented in two or more major browser rendering engines.
- Its defining specification is unlikely to change in ways that will break the web.

Usually, if a technology is supported across several major browsers, the specification will be stable, but this is not always the case.

> And some technologies might have a stable specification and be well-used, but not have native support in browsers ([IMSC](/en-US/docs/Related/IMSC), for example). <!-- need to revisit link -->

Reviewer: Don't know what "not have native support in browsers" means even after going to IMSC. How can something be "well used" but not supported?

I'd also start this sentence with "Note that" ...

Reviewer 2: ```suggestion
On the other hand, some technologies might have a stable specification, but do not have native support in browsers ([IMSC](/en-US/docs/Related/IMSC), for example). <!-- need to revisit link -->
```

Reviewer: @chrisdavidmills ^^^ that suggestion works for me.
